### PR TITLE
Czech language support

### PR DESCRIPTION
**Pull request by killalad**

### CODE REVIEW

File: .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Boris-Em) [inline .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:1]: Do we really need this file?

Comment (killalad) [reply .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata:1]: Well don't know tbh, I just added the new language via Xcode and this was automatically generated with it. Feel free to remove it, if you think it isn't necessary. The only thing I need is just the Czech localisation.

Comment (killalad) [reply .swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata]: done, can we merge now? 🙂

**Review by Boris-Em — APPROVED** [review]

**Comment by wetransferplatform** [comment]
Congratulations! :tada: This was released as part of [Release 1.8.0](https://github.com/WeTransfer/WeScan/releases/tag/1.8.0) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

**Pull request merged**